libs/vkd3d/device.c

```diff
@@ -632,9 +634,10 @@ static const struct vkd3d_debug_option vkd3d_config_options[] =
 
 static void vkd3d_config_flags_init_once(void)
 {
-    const char *config;
+    char config[VKD3D_PATH_MAX];
```

Comment:
Getting a warning

```
libs/vkd3d/device.c#L680
the address of ‘config’ will always evaluate as ‘true’ [-Waddress]
```
Probably somewhere `if (config)` is checked, which would be an invalid check here. I would guess vkd3d_parse_debug_options needs to be updated.

Reply:
Done